### PR TITLE
LPD-23194 Fix facetable logic

### PR DESCRIPTION
**Pull request by riccardo-alberti**
https://liferay.atlassian.net/browse/LPD-23194

### CODE REVIEW

**Comment by riccardo-alberti** [comment]
ci:test:sf

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java [hunk 1]
```diff
@@ -211,8 +211,8 @@ private CPSpecificationOption _updateSpecification(
 			LanguageUtils.getLocalizedMap(specification.getTitle()),
 			LanguageUtils.getLocalizedMap(specification.getDescription()),
 			GetterUtil.getBoolean(
-				cpSpecificationOption.isFacetable(),
-				_isFacetable(specification)),
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java:215]: can we remove the method _isFacetable?

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java (hunk 1 above)
Comment (riccardo-alberti) [reply modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/resource/v1_0/SpecificationResourceImpl.java:215]: yes i'll remove it. thanks

**Comment by riccardo-alberti** [comment]
ci:test:sf

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by riccardo-alberti** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:commerce-functional

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**